test/es-module/test-typescript-commonjs.mjs

```diff
@@ -59,13 +59,26 @@ test('require a .ts file with implicit extension fails', async () => {
 });
 
 test('expect failure of an .mts file with CommonJS syntax', async () => {
-  const result = await spawnPromisified(process.execPath, [
-    fixtures.path('typescript/cts/test-cts-but-module-syntax.cts'),
-  ]);
+  const testFilePath = fixtures.path(
+    'typescript/cts/test-cts-but-module-syntax.cts'
+  );
+  const result = await spawnPromisified(process.execPath, [testFilePath]);
 
-  strictEqual(result.stdout, '');
-  match(result.stderr, /To load an ES module, set "type": "module" in the package\.json or use the \.mjs extension\./);
-  strictEqual(result.code, 1);
+  assert.strictEqual(result.stdout, '');
+
+  const expectedWarning = `Failed to load the ES module: ${testFilePath}. Make sure to set "type": "module" in the nearest package.json file or use the .mjs extension.`;
+
+  assert.match(
+    result.stderr,
+    /Failed to load the ES module:.*test-cts-but-module-syntax\.cts/
+  );
+
+  assert.ok(
+    result.stderr.includes(expectedWarning),
+    `Expected stderr to include: ${expectedWarning}`
+  );
```

Comment:
If you use `assert.ok`,  you should be using try/catch to improve the error output. Can you also add a comment explaining why we need two assertions please?

Reply:
Added, thank you